phpcs.xml.dist

```diff
@@ -13,7 +13,7 @@
 	<arg name="basepath" value="./"/>
```

Comment:
As the file has now been renamed, you'll probably want to add `phpcs.xml` and `.phpcs.xml` (the names of the stnadard supported overload files) to the `.gitignore` file.

Reply:
Thank you!